apps/agent/src/mastra/agents/analyze-github-prs.ts

```diff
@@ -42,4 +42,51 @@ export const analyzePRsAgent = new Agent({
     Return an empty array if no PRs require notifications.
   `,
   model: openai('gpt-4o-mini'),
-})
+});
+
+// currently we are not reading the PR changes, so we can't auto generate the description
+export const suggestPRDescriptionAgent = new Agent({
+  name: 'agent suggest PR description',
+  instructions: `
+    You are an AI assistant tasked with reviewing and improving pull request (PR) titles and descriptions. Your goal is to ensure they are clear, concise, and informative.
+    Title Optimization:
+      Do not generate a specific title, just suggest improvements. Ask the author to improve the title by adding the following:
+      - Follow naming conventions such as [Fix], [Feature], [Refactor]...
+      - Avoid vague or ambiguous titles; specify what is being changed.
+
+    Description Improvements:
+      Do not generate a specific description, just suggest improvements. Ask the author to improve the description by adding the following:
+      - Problem Statement: Clearly define the issue this PR addresses.
+      - Solution: Summarize the changes introduced to fix the issue.
+      - Impact & Risks: Mention any side effects or potential risks.
+      - Testing Steps (if applicable): Provide a clear way to validate the changes.
+      - Remove redundant or vague language.
+
+    Output Format:
+    Return the updated title and description in the following JSON format:
+    \`\`\`
+    {
+      "suggestion_needed": boolean, // true if the PR needs improvement, false otherwise
+      "original_title": "string", // the original title
+      "original_body": "string", // the original description
+      "suggest_title": "string", // the suggested title improvement, empty if no suggestion
+      "suggest_body": "string" // the suggested description improvement, empty if no suggestion
+    }
+    \`\`\`
+
+    Example:
+      User input:
+        Title: "Fix the bug"
+        Description: "I fixed the bug"
+
+      Response:
+      {
+        "suggestion_needed": true,
+        "original_title": "Fix the bug",
+        "original_body": "I fixed the bug",
+        "suggest_title": "Your title is too vague, consider adding details on what is being changed, and apply naming conventions such as [Fix], [Feature], [Refactor]...",
+        "suggest_body": "Your description is too vague, consider adding a problem statement, solution, impact & risks, testing steps (if applicable), and remove redundant or vague language."
+      }
+  `,
+  model: openai('gpt-4o-mini'),
+});
```

Comment:
For the title, please keep the convention as `type(scope?): message`. examples of a valid message:
- feat: add something awesome
- fix(agent): remove annoying typo

For the description, I think we should go with something very generic as checking if the description does include the info for the problem and solution (or todos). If those pieces are missing, then send a remind for the user

Reply:
i've fixed to make the suggestion more generic